gcapi/__init__.py

```diff
@@ -1,5 +1,5 @@
-from gcapi.gcapi import AsyncClient, Client
+from gcapi.gcapi import Client
 
 __author__ = """James Meakin"""
 __email__ = "code@jmsmkn.com"
-__all__ = ["AsyncClient", "Client"]
+__all__ = ["Client"]
```

Comment:
The author and email settings can also be removed, they are covered by pyproject.toml now.